### ISSUE TITLE
64bit support not being picked up by iTunes connect.

### ISSUE THREAD

**Issue by voidstarfire** [libgdx/libgdx#2868]
I have a new version of my app that's uploaded to iTunes connect store and was denied. All apps from the 1st of Feb must be 64BIT.

I'm using the latest version of RoboVM (roboVMVersion = '1.0.0-beta-04') & the latest LibGDX gdxVersion = '1.5.4' which supports real dimensions on all iOS devices. (Huge plus)

I'm using the compilation line of: (In Android Studio)
./gradlew -Probovm.ipaArchs=thumbv7:arm64 build createIPA

Now when using the application loader I get the following email from apple (about 1 minute later):

-----

Dear developer,

We have discovered one or more issues with your recent delivery for "Shorb". Your delivery was successful, but you may wish to correct the following issues in your next delivery:

Missing 64-bit support - Beginning on February 1, 2015 new iOS apps submitted to the App Store must include 64-bit support and be built with the iOS 8 SDK. Beginning June 1, 2015 app updates will also need to follow the same requirements. To enable 64-bit in your project, we recommend using the default Xcode build setting of “Standard architectures” to build a single binary with both 32-bit and 64-bit code.

After you’ve corrected the issues, you can use Xcode or Application Loader to upload a new binary to iTunes Connect.

Regards,

The App Store team

-----

PS Is there any way of checking that it's 64bit before uploading, otherwise i'm going to end up with gazillion iOS builds.

**Comment by badlogic** [comment]
What was the problem?

**Comment by voidstarfire** [comment]
I forgot to update my dependencies with:        
classpath 'org.robovm:robovm-gradle-plugin:1.0.0-beta-04' //Remember to update var.

You can still use
roboVMVersion = '1.0.0-beta-04' 

Even if the dependancy is miss matched, it will revert.

**Comment by dannysperling** [comment]
We're running into this issue ourselves, but that didn't fix our problems. We've added the classpath to our build, but we still don't see a choice to do 64-bit when the pop-up shows up (we've re-built with gradle, both `build` and `eclipse`, and restarted eclipse and reimported the projects). We continue to get rejected with the same message.

We've needed to use the pop-up instead of build from the commandline because the commandline wasn't picking up our provisioning profile. If there's an easy way to get it to recognize the provisioning profile, that would be helpful.

Any advice on how we should proceed?
